apps/codesandbox-react-northstar-template/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "codesandbox-react-northstar-template",
   "description": "Template for creating codesandbox environment for @fluentui/react-northstar packages.",
-  "version": "7.0.0",
+  "version": "1.0.0",
```

Comment:
"reset" the package version to be `1.0.0` because
1. it's not related to fluentui/react
2. it's private / never published